<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>78</x>
      <y>10</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>IP地址：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>78</x>
      <y>60</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>端口：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ip">
    <property name="geometry">
     <rect>
      <x>148</x>
      <y>20</y>
      <width>301</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="port">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>70</y>
      <width>301</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="connect_bt">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>20</y>
      <width>141</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>连接服务器端</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="display_message">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>110</y>
      <width>711</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="text_input">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>460</y>
      <width>511</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="send_message">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>460</y>
      <width>93</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>发送消息</string>
    </property>
   </widget>
   <widget class="QPushButton" name="disconnect_bt">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>460</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>断开连接</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
